src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.stopwatch {
+  display: flex;
+  position: relative;
+  justify-content: center;
+  align-items: center;
+  height: 80vmin;
+  width: 80vmin;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    top: 80vmin / 2 - 20vmin;
```

Comment:
The calculation 'top: 80vmin / 2 - 20vmin;' might not work as intended because division and subtraction operations are not grouped. Consider using parentheses to ensure the correct order of operations: 'top: (80vmin / 2) - 20vmin;'.